db/schema.rb

```diff
@@ -117,8 +114,8 @@
   create_table "host_genomes_metadata_fields", id: false, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
     t.bigint "host_genome_id", null: false
     t.bigint "metadata_field_id", null: false
-    t.index ["host_genome_id", "metadata_field_id"], name: "index_host_genomes_metadata_fields"
-    t.index ["metadata_field_id", "host_genome_id"], name: "index_metadata_fields_host_genomes"
+    t.index ["host_genome_id", "metadata_field_id"], name: "index_host_genomes_metadata_fields", unique: true
```

Comment:
Not sure what happened here... what's the best practice for resetting to the master branch migration before applying my own?

Reply:
Sorry, no best practice yet :) This has been a recurring that we need to solve, probably with the initial db config.

You might have to remove it by hand... or reset and copy over.

Reply:
To be clear, I should just copy over the lines related to my new tables and ignore any changes to other tables?

Reply:
Those are enough, yes. You can leave the timestamp on top too.